<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>俄罗斯方块</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Widget{background-color: rgb(0, 153, 230);}</string>
  </property>
  <widget class="QPushButton" name="btn_start">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>530</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:3px;
color: rgb(0, 20, 20);
background-color: rgb(140, 255, 155);</string>
   </property>
   <property name="text">
    <string>开始游戏</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_score">
   <property name="geometry">
    <rect>
     <x>316</x>
     <y>140</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文新魏</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>得分：0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_time_sys">
   <property name="geometry">
    <rect>
     <x>316</x>
     <y>30</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文新魏</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>系统时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_time_game">
   <property name="geometry">
    <rect>
     <x>316</x>
     <y>85</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文新魏</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>游戏时间:0分0秒</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_score_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>240</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文新魏</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>B22080228 徐基恒</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_score_3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>270</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文新魏</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>B22080204 欧阳芳菲</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_score_4">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>300</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文新魏</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>B22080214 祁文睿</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_score_5">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>210</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文新魏</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>小组成员</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
